src/app/components/time/time.component.ts

```diff
@@ -11,10 +11,10 @@ export class TimeComponent implements OnInit {
 
   currentTime$: ITime;
 
-  constructor(private data: TimeService) { }
+  constructor(private _timeService: TimeService) { }
```

Comment:
We don't usually use the underscore convention for private variables in our projects.
The IDE does a good job in highlighting the different types.